ClosedXML/Excel/Ranges/XLRangeAddress.cs

```diff
@@ -24,6 +24,11 @@ public static XLRangeAddress EntireRow(XLWorksheet worksheet, int row)
                 new XLAddress(worksheet, row, XLHelper.MaxColumnNumber, false, false));
         }
 
+        public static XLRangeAddress Invalid = new XLRangeAddress(
```

Comment:
That's great, but the field should not be editable. So it should either be `readonly`, or be a property without a setter, or be an expression `public static XLRangeAddress Invalid => new XLRangeAddress(...`.

Reply:
And I guess we have to do something with the equality of invalid addresses. Comparing two invalid range addresses must always return the same result: either true or false. I am not sure which has more sense but it should definitely be consistent. Now the result may be any. Shall I open a new issue for this?

Reply:
This is an internal struct field, so I wasn't too catious, but I've made it read-only anyway.

Reply:
> And I guess we have to do something with the equality of invalid addresses. Comparing two invalid range addresses must always return the same result: either true or false. I am not sure which has more sense but it should definitely be consistent. Now the result may be any. Shall I open a new issue for this?

It's OK, I'll fix this quickly.

Reply:
I guess we should do the same (new static field and equality operator) for `XLAddress` too?

Reply:
Agree. I even thought we already have this but it seems we don't

Reply:
So we agree that 2 invalid addresses/range addresses are equal if both are invalid (regardless of their actual row/column numbers)?

Do you think 2 invalid addresses are equal is they are both invalid, but on different worksheets? I would argue not equal.

Reply:
Also, I see our current implementation of `XLAddress.Equals()` implies that `Sheet1!A1` is equal to `Sheet2!A1`. Worksheet is ignored. Maybe that should change? I can see valid reasons either way.

Reply:
Addresses on different worksheets are not equal, to my opinion

Reply:
> Also, I see our current implementation of XLAddress.Equals() implies that Sheet1!A1 is equal to Sheet2!A1. Worksheet is ignored. Maybe that should change?

I don't remember why this was needed to compare addresses from different worksheets, but for me, consider worksheets during comparison also seems reasonable. We also have a method `WithoutWorksheet` that can be utilized to compare addresses regardless of their worksheets - perhaps, we could do it public.

Reply:
Ok. I think we must make `XLRangeAddress.IsValid` and `XLAddress.IsValid` public too.

Reply:
Maybe this should be a new issue after all, as you suggested. Are you OK with the rest of this PR as is?

Reply:
Yes, it's fine